src/classes/OPP_OpportunityContactRoles_TDTM.cls

```diff
@@ -762,7 +776,9 @@ public without sharing class OPP_OpportunityContactRoles_TDTM extends TDTM_Runna
 
         for (Opportunity eachOppty : currentOpportunities) {
             if (eachOppty.AccountId != null
-                && OPP_AutomatedSoftCreditsService.isOrganizationalAccount(oppAccountDetails.get(eachOppty.AccountId).npe01__SYSTEMIsIndividual__c)) {
+                && OPP_AutomatedSoftCreditsService.isOrganizationalAccount(oppAccountDetails.get(eachOppty.AccountId).npe01__SYSTEMIsIndividual__c)
```

Comment:
Same question here about checking for `eachOppty.DisableContactRoleAutomation__c` after the `for` and using `continue`.

Reply:
The check for `Disable Contact Role Automation` is another check as verification the AccountId is not null, so I would treat them the same. Please let me know if that is not the case.